.github/workflows/go.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17
+        go-version: 1.18
```

Comment:
curious why you changed the version here when `go.mod` still references 1.17?

Reply:
I just updated the gitlab runner version, older version of go.mod are still compatible with new versions :)